### PR TITLE
CT-1287 wire pause/resume methods

### PR DESCRIPTION
**Pull request by aacotroneo**

### CODE REVIEW

File: PublisherView.js [hunk 1]
```diff
@@ -81,6 +81,12 @@ class PublisherView extends React.Component {
   sayHello() {
     OpenTokPublisherViewManager.sayHello();
   }
```
Comment (ghost) [inline PublisherView.js]: Does this need to be removed?

Comment (aacotroneo) [reply PublisherView.js]: yeah - we should! it does work (it's been merged on the native lib side).. was mainly used for testing/debugging all the native bridge.
I'll remove this part now

Comment (aacotroneo) [reply PublisherView.js]: sorry.. this is the native lib.. I should remove it from our code! I think it may be around :)

Comment (ghost) [reply PublisherView.js]: Yeah, feel free to remove it here and we can remove it from the POC in another PR there.

**Review by ghost — APPROVED** [review]
+1

**Pull request merged**